.changeset/fifty-squids-work.md

```diff
@@ -0,0 +1,5 @@
+---
+"@localyze-pluto/components": minor
```

Comment:
Why does this need to be a minor?

Reply:
I was thinking about the impact on the docs